### ISSUE TITLE
klab parse

### ISSUE THREAD

**Issue by asymmetric** [dapphub/klab#308]
Add a command that makes use of https://github.com/kframework/k/pull/1040, i.e. that invokes `kprove --skip-invoking-backend`, which can be run in CI to just check for parsing errors quickly.

**Comment by MrChico** [comment]
Is this integrated in the kevm we are using?

**Comment by asymmetric** [comment]
@MrChico I've cherry picked the changes [in this branch](https://github.com/kframework/evm-semantics/compare/master...dapphub:skipper), but we really should catch up with upstream.

**Comment by asymmetric** [comment]
Well it needs to be merged into K before we can/should use it, so yes, this is still blocked.

**Comment by asymmetric** [comment]
Steps:

1. Merge https://github.com/kframework/k/pull/1040 into K
1. Bump evm-semantics in klab
1. Add `klab parse`
1. Add "quick" scheduler to buildbot
